pkg/controller/workload/workload_processor.go

```diff
@@ -1022,3 +1052,30 @@ func (p *Processor) storeWorkloadPolicies(uid string, polices []string) {
 		log.Errorf("storeWorkloadPolicies failed, workload %s, err: %s", uid, err)
 	}
 }
+
+func (p *Processor) deleteOutdataFrontend(addresses [][]byte) error {
+	frontKey := bpf.FrontendKey{}
+	for _, address := range addresses {
+		nets.CopyIpByteFromSlice(&frontKey.Ip, address)
+		if err := p.bpf.FrontendDelete(&frontKey); err != nil {
```

Comment:
In case it delete a service frontend map i think we should do some protection. There are some other work we can do, maybe you should do that in another pr of you